### PR TITLE
added new crew role Mole

### PR DESCRIPTION
**Pull request by ryuk201198**

### CODE REVIEW

**Comment by ryuk201198** [comment]
Merge this after the full release

**Comment by ryuk201198** [comment]
Please review

**Review by Tommy-XL — APPROVED** [review]

File: Roles/Crewmate/Mole.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using System.Linq;
+using UnityEngine;
+using static TOHE.Options;
+
+namespace TOHE.Roles.Crewmate
+{
+    public static class Mole
+    {
+        private static readonly int Id = 26000;
+        private static List<byte> playerIdList = new();
```
Comment (Tommy-XL) [inline Roles/Crewmate/Mole.cs:11]: I think that `playerIdList` can be removed, since it is not used in the code

Comment (ryuk201198) [reply Roles/Crewmate/Mole.cs]: > I think that `playerIdList` can be removed, since it is not used in the code

done

**Pull request merged**